C/source/generateHologram.cu

```diff
@@ -37,8 +39,18 @@
 #define BLOCK_SIZE 256
 #define BLOCK_STRIDE 4
 
-#define SLM_HEIGHT 1440
-#define SLM_WIDTH 2560
+#ifdef DEFAULT_HEIGHT
+	#define SLM_HEIGHT DEFAULT_HEIGHT	//Declaring SLM_HEIGHT as DEFAULT_HEIGHT from global_module_defs.hpp
+#else
+	#define SLM_HEIGHT 1440	
+#endif	
+
+#ifdef DEFAULT_WIDTH
+	#define SLM_WIDTH DEFAULT_WIDTH		//Declaring SLM_WIDTH as DEFAULT_WIDTH from global_module_defs.hpp
+#else	
+	#define SLM_WIDTH 2560
+#endif
+	
```

Comment:
Change the names `DEFAULT_HEIGHT` and `DEFAULT_WIDTH` to match the names used to address @mhuzai 's comment in PR [#162](https://github.com/ILLIXR/ILLIXR/pull/162)

Reply:
@e3m3 I don't understand, but i guess #define should not but used here but it should be like this
```
#ifdef DEFAULT_WIDTH
     SML_WIDTH=DEFAULT_WIDTH
#else
```
If yes, then allow me to make these changes. Thanks :)